### PR TITLE
Completed css-flexbox assignment

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image

image

image

image

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        display: flex;
+        justify-content: center;
+        min-height: 100vh;
+        align-items: center;
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'card-wrapper' id ... Opening curly bracket*/
+      .card-wrapper {
+        flex-basis: calc(100% / 3);
+        justify-content: center;
```
Comment (shawnkost) [inline css-flexbox/align-items.html:23]: `justify-content` is a parent property which means it gets applied on elements that also have `display: flex`. In this case, it should be removed

File: css-flexbox/align-items.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'pokemon-card' id ... Opening curly bracket*/
+      .pokemon-card {
+        padding: 10px;
+        margin: 7px;
```
Comment (shawnkost) [inline css-flexbox/align-items.html:29]: Not necessary

File: css-flexbox/display-flex.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      /* A new a new css rule set selecting all elements in the 'card-deck' id ... Opening curly bracket*/
+      #card-deck {
+        display: flex;
+        flex-direction: flex;
+        flex: row;
```
Comment (shawnkost) [inline css-flexbox/display-flex.html:16]: I don't believe this exists. `flex` expects numerical values or `initial, auto, none`. I would remove it.

File: css-flexbox/display-flex.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      /* A new a new css rule set selecting all elements in the 'card-deck' id ... Opening curly bracket*/
+      #card-deck {
+        display: flex;
+        flex-direction: flex;
```
Comment (shawnkost) [inline css-flexbox/display-flex.html:15]: This should either be `row` or `column`. By default it's `row`, so you can remove this

File: css-flexbox/display-flex.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        flex: row;
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'pokemon-card' id ... Opening curly bracket*/
+      .pokemon-card {
+        padding: 10px;
```
Comment (shawnkost) [inline css-flexbox/display-flex.html:21]: The `padding` should be getting applied to the `card-wrapper`

File: css-flexbox/display-flex.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        margin: 7px;
```
Comment (shawnkost) [inline css-flexbox/display-flex.html:22]: Not needed

File: css-flexbox/flex-wrap.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'card-wrapper' id ... Opening curly bracket*/
+      .card-wrapper {
+        flex-basis: calc(100% / 6);
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'pokemon-card' id ... Opening curly bracket*/
+      .pokemon-card {
+        padding: 10px;
```
Comment (shawnkost) [inline css-flexbox/flex-wrap.html:25]: `padding` should be getting applied to the `card-wrapper`

File: css-flexbox/flex-wrap.html [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+        margin: 7px;
```
Comment (shawnkost) [inline css-flexbox/flex-wrap.html:26]: Not needed

File: css-flexbox/justify-content.html [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      /* A new a new css rule set selecting all elements in the 'card-deck' id ... Opening curly bracket*/
+      #card-deck {
+        display: flex;
+        flex: row;
```
Comment (shawnkost) [inline css-flexbox/justify-content.html:15]: Don't believe this value exists

File: css-flexbox/justify-content.html [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+      /* A new a new css rule set selecting all elements in the 'card-wrapper' id ... Opening curly bracket*/
+      .card-wrapper {
+        flex-basis: calc(100% / 4);
+        justify-content: center;
+      } /* Closing curly bracket*/
+
+      /* A new a new css rule set selecting all elements in the 'pokemon-card' id ... Opening curly bracket*/
+      .pokemon-card {
+        padding: 10px;
```
Comment (shawnkost) [inline css-flexbox/justify-content.html:27]: `padding` should be getting applied to the `card-wrapper`

File: css-flexbox/justify-content.html [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+        margin: 7px;
```
Comment (shawnkost) [inline css-flexbox/justify-content.html:28]: Not needed

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Aiden!

File: css-flexbox/align-items.html (hunk 1 above)
Comment (Aiden-Peace300) [reply css-flexbox/align-items.html:23]: What a silly mistake thank you!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**